content/javascript/concepts/iterators/iterators.md

```diff
@@ -11,19 +11,15 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Iterators are used to loop over a group of data members, or a collection.
+**Iterators** are used to loop over a group of data members, or a collection.
 
-An iterator is an object that implements the iteration protocols. Many built-in data types (Strings, Arrays, Maps, Sets, etc.) have an iterator property that make them iterable.
+An iterator is an object that implements the iteration protocols. Many built-in data types ([strings](https://www.codecademy.com/resources/docs/javascript/strings), [arrays](https://www.codecademy.com/resources/docs/javascript/arrays), [maps](https://www.codecademy.com/resources/docs/javascript/map), [sets](https://www.codecademy.com/resources/docs/javascript/sets), etc.) have an iterator property that make them iterable.
 
-## Iterable Protocol
+## Iterable and Iterator Protocols
 
-All iterable objects implement the `@@iterator` method. In other words, an object must have or inherit, via its prototype chain, the `@@iterator` property key.
+All _iterable objects_ implement the `@@iterator` method. In other words, an object must have or inherit, via its prototype chain, the `@@iterator` property key. When an object is to be iterated, the `@@iterator` method is called without any arguments, and the returned iterator obtains the values or elements to be looped through.
 
-When an object is to be iterated, the `@@iterator` method is called without any arguments, and the returned iterator obtains the values or elements to be looped through.
-
-## Iterator Protocol
-
-The iterator protocol, by definition, implements the `next()` method and returns an object with at least two properties:
+The _iterator protocol_, by definition, implements the `next()` method and returns an object with at least two properties:
 
 - `done` is a boolean that determines whether the sequence has been completed or consumed. Its value is `false` if incomplete and `true` otherwise.
 - `value` is any type of value the iterator returns.
```

Comment:
The last example for this entry says it returns "{ value: 3, done: true }" but I don't get that line when I run it 🤔

Reply:
Sorry! I think that was leftover from `console.log(..)` debugging lol 😆 I'll remove it.